test/framework/k8s_cluster.go

```diff
@@ -1027,6 +1027,34 @@ func (c *K8sCluster) DeleteMesh(mesh string) error {
 	return err
 }
 
+func (c *K8sCluster) GetClusterIP(serviceName, namespace string) (string, error) {
+	service, err := k8s.GetServiceE(
+		c.t,
+		c.GetKubectlOptions(namespace),
+		serviceName,
+	)
+	if err != nil {
+		return "", err
+	}
+	return service.Spec.ClusterIP, nil
+}
+
+func (c *K8sCluster) GetLBIngressIP(serviceName, namespace string) (string, error) {
+	service, err := k8s.GetServiceE(
+		c.t,
+		c.GetKubectlOptions(namespace),
+		serviceName,
+	)
+	if err != nil {
+		return "", err
+	}
+	ingress := service.Status.LoadBalancer.Ingress
+	if len(ingress) == 0 {
+		return "", errors.New("ingress not found")
```

Comment:
The wording here can be improved since it's easy to be confused since "ingress" has multiple meanings in many scenarios. At here we can't find any ingress for the load balancer service, so it's better to make it `fmt.Sprintf("ingress information not found on the load balancer service '%s'", serviceName)`